apps/ledger-live-mobile/src/components/accountSyncRefreshControl.tsx

```diff
@@ -24,6 +25,7 @@ export default <P,>(ScrollListLike: React.ComponentType<P>) => {
     const [refreshing, setRefreshing] = useState(false);
 
     function onPress() {
+      track("buttonClicked", { button: "pull to refresh" });
```

Comment:
I thought about it but as there's no dedicated file I didn't do it.
But it might be a good thing to have a file that groups all the track event hooks, as we often use track(“button_clicked”) and others.

Reply:
exactly, this makes it confusing to have multiple selectors that semantically have the same meaning but just a different syntax. 

However, agree this may be outside the scope of your PR.